flight_phase_files/HALO_RF04_20200126_info.yaml

```
@@ -21,71 +26,366 @@ remarks:
+  start: 2020-01-26 12:26:08
+  end: 2020-01-26 15:39:13
+  dropsondes:
+    GOOD:
+    - HALO-0126_s01
+    - HALO-0126_s02
+    - HALO-0126_s03
+    - HALO-0126_s04
+    - HALO-0126_s05
+    - HALO-0126_s06
+    - HALO-0126_s07
+    - HALO-0126_s08
+    - HALO-0126_s09
+    - HALO-0126_s10
+    - HALO-0126_s11
+    - HALO-0126_s12
+    - HALO-0126_s14
+    - HALO-0126_s15
+    - HALO-0126_s16
+    - HALO-0126_s17
+    - HALO-0126_s18
+    - HALO-0126_s20
+    - HALO-0126_s21
+    - HALO-0126_s22
+    - HALO-0126_s23
+    - HALO-0126_s24
+    - HALO-0126_s25
+    - HALO-0126_s26
+    - HALO-0126_s27
+    - HALO-0126_s28
+    - HALO-0126_s29
+    - HALO-0126_s30
+    - HALO-0126_s31
+    - HALO-0126_s32
+    - HALO-0126_s33
+    - HALO-0126_s34
+    - HALO-0126_s36
+    BAD:
+    - HALO-0126_s19
+    UGLY:
+    - HALO-0126_s13
+    - HALO-0126_s35
+- kinds:
+  - straight_leg
+  name: Leg-1 over Meteor
+  irregularities:
+  - slightly high roll towards end of the leg
+  segment_id: HALO-0126sl1
   start: 2020-01-26 15:50:46
   end: 2020-01-26 16:04:52
-- kind: straight_leg
-  name: leg2_over_Meteor
+  dropsondes:
+    GOOD: []
+    BAD: []
+    UGLY:
+    - HALO-0126_s37
+- kinds:
+  - straight_leg
+  name: Leg-2 over Meteor
+  irregularities:
+  - slightly high roll at start of leg
+  segment_id: HALO-0126sl2
   start: 2020-01-26 16:11:14
   end: 2020-01-26 16:31:14
-- kind: straight_leg
-  name: leg3_over_Meteor
+  dropsondes:
+    GOOD: []
+    BAD: []
+    UGLY: []
+- kinds:
+  - straight_leg
+  name: Leg-3 over Meteor
+  irregularities: []
+  segment_id: HALO-0126sl3
   start: 2020-01-26 16:37:21
   end: 2020-01-26 16:52:17
-- kind: straight_leg
-  name: leg4_over_Meteor
+  dropsondes:
+    GOOD: []
+    BAD: []
+    UGLY: []
+- kinds:
+  - straight_leg
+  name: Leg-4 over Meteor
+  irregularities: []
+  segment_id: HALO-0126sl4
   start: 2020-01-26 16:58:46
   end: 2020-01-26 17:14:14
-- kind: radar_calibration_wiggle
-  name: radar_cal_wiggle
+  dropsondes:
+    GOOD:
+    - HALO-0126_s38
+    - HALO-0126_s39
+    BAD: []
+    UGLY: []
+- kinds:
+  - radar_calibration_wiggle
+  name: Radar Calibration Wiggle
+  irregularities: []
+  segment_id: HALO-0126rcw1
   start: 2020-01-26 17:14:14
-  end: 2020-01-26 17:21:01
-- kind: circle
-  name: c4
-  start: 2020-01-26 17:21:01
-  end: 2020-01-26 18:12:50
-- kind: circle_break
-  name: cb4to5
-  start: 2020-01-26 18:12:50
-  end: 2020-01-26 18:36:01
-- kind: circle
-  name: c5
-  start: 2020-01-26 18:36:01
-  end: 2020-01-26 19:29:21
-- kind: circle_break
-  name: cb5to6
-  start: 2020-01-26 19:29:21
-  end: 2020-01-26 19:41:01
-- kind: circle
-  name: c6
-  start: 2020-01-26 19:51:32
-  end: 2020-01-26 20:45:42
-- kind: lidar_calibration
-  name: lidar_cal
+  end: 2020-01-26 17:16:15
+  dropsondes:
+    GOOD: []
+    BAD: []
+    UGLY: []
+- kinds:
+  - circle
+  name: Circle-4
+  irregularities: []
+  segment_id: HALO-0126c4
+  start: 2020-01-26 17:19:54
+  end: 2020-01-26 18:18:34
```

This circle overlaps. The overlap can be reduced by ending the circle earlier.
```suggestion
  end: 2020-01-26 18:17:54
```